facebook/facebook-sdk-for-unity
# `bug` Fixed Build Issue and upload path for IOS

Thanks for proposing a pull request!

To help us review the request, please complete the following:


## Pull Request Details
1. I removed the method "IOSFBUpdateUserProperties" Since it caused issues when trying to build in xcode, since that method no longer existed on the IOS plugin. 
2. I changed the path for uploading images and videos from AbsolutePath to LocalPath. This was necessary because on a real device, the image could not be loaded from the AbsolutePath path. Changing this to LocalPath fixed the issue. Especially if it was an image saved from unity to that path.

Describe what you accomplished in this pull request (for example, what happens before the change, and after the change)

## Test Plan

Test Plan: 
1. Build Plugin
2. Implement in Unity Project.
3. Make IOS build
4. Make sure it is successful
5. Trying to using the facbook gaming services to share an image screenshot from unity.
